enrollment.js

```diff
@@ -181,9 +225,33 @@ addInputHandler('enr_pn', function(input){ //enr phone number step
     get_time();
 });
 
-addInputHandler('enr_glus', function(input){ //enr group leader / umudugudu support id step. last registration step
+addInputHandler('enr_glus',function(input){
+    
+    input = input.replace(/\W/g,'');
+    state.vars.current_step = 'enr_glus';
+    
+    if(input == 99){
+        sayText(msgs('exit', {}, lang));
+        stopRules();
+        return null;
+    }
+    else{
+        state.vars.glus = input;
+        var splash_menu = populate_menu('enr_group_id_confirmation', lang, 300);
```

Comment:
We don't need to use the populate menu function since the menu options don't change; this will simplify the code.